components/admin/db/seeds.rb

```diff
@@ -36,8 +36,9 @@ def initialize_taxonomies!
       'Skills to Learn',
       'Skills Needed',
     ].map(&:to_sym)
-    taxonomy_names.each do |taxonomy_name|
-      Taxonomy.find_or_create_by!(name: taxonomy_name)
+    taxonomy_weights = [1, 2, 2]
+    taxonomy_names.zip(taxonomy_weights).each do |taxonomy_name, weight|
+      Taxonomy.find_or_create_by!(name: taxonomy_name, weight: weight)
```

Comment:
This is unnecessary and should probably be built as an attribute hash, then the names extracted from it.